relay-general/src/store/normalize.rs

```diff
@@ -865,6 +892,10 @@ pub fn light_normalize_event(
         ); // Measurements are part of the metric extraction
         normalize_breakdowns(event, config.breakdowns_config); // Breakdowns are part of the metric extraction too
 
+        if event.ty.value() == Some(&EventType::Transaction) {
+            normalize_app_start_spans(event);
+        }
```

Comment:
```suggestion
```

Could you please move this into the `if config.light_normalize_spans` block? "Light" normalization is only necessary for metrics extraction, which is currently still feature-flagged for spans.